dev-tools/mage/settings.go

```diff
@@ -248,8 +251,10 @@ func findElasticBeatsDir() (string, error) {
 
 	const devToolsImportPath = elasticBeatsImportPath + "/dev-tools/mage"
 
-	// Search in project vendor directories.
+	// Search in project vendor directories. Order is relevant
 	searchPaths := []string{
+		// beats directory of apm-server
+		filepath.Join(repo.RootDir, "_beats", "dev-tools", "packaging"),
```

Comment:
Personally I think `_beats/dev-tools/packaging` is more readable and recognizable as a path. `Join` cleans the path so the OS specific slashes don’t matter.